archunit/src/test/java/com/tngtech/archunit/testutil/JavaCallQuery.java

```diff
@@ -0,0 +1,63 @@
+package com.tngtech.archunit.testutil;
+
+import java.util.Set;
+
+import com.google.common.base.Predicate;
+import com.google.common.collect.FluentIterable;
+import com.tngtech.archunit.core.domain.JavaCodeUnit;
+import com.tngtech.archunit.core.domain.JavaMethod;
+import com.tngtech.archunit.core.domain.JavaMethodCall;
+
+public class JavaCallQuery {
+
+    private FluentIterable<JavaMethodCall> calls;
+
+    private JavaCallQuery(Set<JavaMethodCall> calls) {
+        this.calls = FluentIterable.from(calls);
+    }
+
+    public static JavaCallQuery methodCallTo(final JavaMethod method) {
```

Comment:
I usually wouldn't mix a static method between fields / constructors / instance methods.
If that's the recommended entry point, why not put it at the very top of the class?

Reply:
Yes, agreed, but I put it on the bottom, because I think that's more consistent within the code base (we have many other places where factory methods are at the bottom, but I don't recall one where they are on the top). Of course I can also see why it makes sense to put it on the top when it has constructor semantics, but I want to keep it consistent here...